glass/grf/_transformations.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+# type imports for NDArray[Any] cannot be moved into TYPE_CHECKING here
+# otherwise, the dispatch mechanism cannot resolve the class dynamically
+from typing import Any
+
+from numpy.typing import NDArray  # noqa: TC002
+
+from glass.grf._core import corr, dcorr, icorr
+
+
+@dataclass
+class Normal:
+    """
+    Transformation for normal fields.
+
+    .. math::
+
+       t(X) = X
+
+    This is the identity transformation.
+
+    """
+
+    def __call__(self, x: NDArray[Any], _var: float, /) -> NDArray[Any]:
+        """Return *x* unchanged."""
+        return x
+
+
+@dataclass
+class Lognormal:
+    r"""
+    Transformation for lognormal fields.
+
+    .. math::
+
+       t(X) = \lambda \bigl[\exp(X - \tfrac{\sigma^2}{2}) - 1\bigr]
+
+    The "lognormal shift" parameter :math:`\lambda` is the scale
+    parameter of the distribution, with determines the smallest
+    possible value :math:`-\lambda` of the lognormal field.
+
+    """
+
+    lamda: float = 1.0
+
+    def __call__(self, x: NDArray[Any], var: float, /) -> NDArray[Any]:
+        """Transform *x* into a lognormal field."""
+        xp = x.__array_namespace__()
+        x = xp.expm1(x - var / 2)
+        if self.lamda != 1.0:
+            x = self.lamda * x
+        return x
+
+
+@dataclass
+class SquaredNormal:
+    r"""
+    Transformation for squared normal fields as introduced by
+    [Tessore25]_.
```

Comment:
`Tessore25` link doesn't work.

Reply:
Looks fine in there rendered docs?
![Untitled](https://github.com/user-attachments/assets/c9a17f55-e2dd-48e6-8ae1-6da72fe0758a)

Reply:
It renders but `[Tessore23]` works as a hyperlink, `[Tessore25]` does not

Reply:
Is that maybe just because the one reference takes you to a different page, the other reference is right there? In #497, there's a separate "bibliography" page where this link here will go.

Reply:
Yeah, maybe. I guess we can leave it as it is and hope it's fixed in #497? Probably worth raising an issue so we remember to check.